guide/english/python/itertools/index.md

```diff
@@ -58,4 +58,15 @@ combinations('ABCD', 2)
 combinations_with_replacement('ABCD', 2)	 	                                                     AA AB AC AD BB BC BD CC CD DD
 
 
-Source:https://docs.python.org/3/library/itertools.html
\ No newline at end of file
+Source:https://docs.python.org/3/library/itertools.html
+
+##Changes
+
+###compress()
+It make an iterator that filters elements from data returning only those that have a corresponding element in selectors that evaluates to True. Stops when either the data or selectors iterables has been exhausted.
+
+import itertools as i
```

Comment:
import itertools as i
 i.compress('ABCDEF', [1,0,1,0,1,1]) 
 --> A C E F

Please use correct formatting for posting code samples and output. Use the [guide formatting](https://raw.githubusercontent.com/freeCodeCamp/freeCodeCamp/8e8ea8cbdffc7f8bcf2c846824993c46ff8b0ae9/guide/english/python/itertools/index.md) as a reference.

Reply:
Thanks for the suggestion. I Will keep in mind next time.